tests/websocket-clients/curl-client-json.sh

```diff
@@ -0,0 +1 @@
+curl -X POST http://localhost:8080/api/postKVP -H "Content-Type: application/octet-stream" --data-binary "@example.json"
```

Comment:
if you remove -> `-H "Content-Type: application/octet-stream"` does it still work.. I'm not suggesting changing it just wanted to know if it will work.

you also need to add another send for one binary to ensure binary is working, but you also need to test sending the json content-type and also the yaml content-type

```c
curl -X POST -H "Content-Type: application/json" --data @your_file.json <API_ENDPOINT>
```

Reply:
yes, it works. I have updated these script files and added a new one for binary.